### ISSUE TITLE
organizing the A1112 exception of glyf test failure by failure sub-types.

### ISSUE THREAD

**Issue by HinTak** [Microsoft/Font-Validator#40]
The A1112 exception issues filed so far is organized currently by which font files on which it happens.
It would be useful to organize by failure sub-types across the font collections to tick them off as each failure sub-types are fixed.

https://github.com/Microsoft/Font-Validator/issues/39
https://github.com/Microsoft/Font-Validator/issues/37
https://github.com/Microsoft/Font-Validator/issues/36
https://github.com/Microsoft/Font-Validator/issues/35

**Comment by HinTak** [comment]
okay, across the three current major platforms and the roughly 10 font files mentioned, there are only 3 kinds of A1112 exceptions:

```
ValidateSimpContInters Exception: Exception GMath: class: Bez2D method: ParamFromSupport details: NOT IMPLEMENTED
ValidateSimpContInters Exception: Exception GMath: class: Equation method: RootsAll details: Degree 3
ValidateSimpContMisor Exception: Exception Glyph: class: Glyph method: ValidateSimpContMisor
```

**Comment by HinTak** [comment]
Fixed in upcoming v2.1
